editors/code/src/config.ts

```diff
@@ -25,10 +21,7 @@ export interface CargoFeatures {
     loadOutDirsFromCheck: boolean;
 }
 
-export const enum UpdatesChannel {
-    Stable = "stable",
-    Nightly = "nightly"
-}
+export type UpdatesChannel = "stable" | "nightly";
```

Comment:
Good thing that TypeScript allows for *stringly-typed* apis, though I prefer enums, but no too strong arguments.

Reply:
Yep, makes sense, will update tomorrow